sqlglot/parser.py

```diff
@@ -681,6 +687,7 @@ class Parser(metaclass=_Parser):
         "TITLE": lambda self: self.expression(
             exp.TitleColumnConstraint, this=self._parse_var_or_string()
         ),
+        "TTL": lambda self: self._parse_ttl(),
```

Comment:
It's how I originally had it, I just made it a function so that both `PROPERTY_PARSERS` and `CONSTRAINT_PARSERS` can refer to the same implementation and avoid changing >1 sections of code for this later. If there's no issue with that happy to revert to a lambda.